ext/msgpack/unpacker.h

```diff
@@ -49,7 +47,7 @@ struct msgpack_unpacker_stack_t {
     msgpack_unpacker_stack_t *parent;
 };
 
-#define MSGPACK_UNPACKER_STACK_SIZE (8+4+8+8)  /* assumes size_t <= 64bit, enum <= 32bit, VALUE <= 64bit */
```

Comment:
I think we can get rid of this definition. It doesn't seem to be used anywhere.

Reply:
Oops. Fixed.